Shared/AppFeature/GridFeature/GridCellFeature/GridCellState.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
```

Comment:
PLEASE UPDATE ALL files to use 4 char spacers
Format and save.
All files please